src/testsuite/arith128_test_i128.c

```diff
@@ -4764,11 +4764,192 @@ test_setbq (void)
 }
 
 //#define __DEBUG_PRINT__ 1
+int
+test_cmpuq_p2 (void)
```

Comment:
A new function isn't expected for a patch that purports to change a return type.  Is this testing new functionality?

Reply:
Sorry. Fell into the GCC 6 ICE associated with large vector functions. GCC will not fix frozen GCC release. This is what I have to do to get it to compile.